internal/db/schema/migrations/oss/postgres/31/01_credential_type.up.sql

```diff
@@ -0,0 +1,22 @@
+begin;
+
+  -- drop constraint so we can migrate user_password to username_password
```

Comment:
can you add a comment to note that this replaces the constraint defined in `2/02_credential_type.up.sql` and likewise add a comment in `2/02_credential_type.up.sql` to note that the constraint is replaced in `31/01_credential_type.up.sql`